client/src/components/Jobs/Jobs.scss

```diff
@@ -1,61 +1,73 @@
-.jobs-view {
-  /* icon */
-  th:nth-child(1),
-  td:nth-child(1) {
-    min-width: 2em;
+/* Dashboard Table */
```

Comment:
The introduction of `table-layout: fixed` made `min-width` ineffectual, so I used percentages (that add to 100%), and I changed those percentages depending on screen width. Also, I simplified the syntax (because I needed to add so much code). Not every column needs a percentage, but I lost to my OCD.